ibis/impala/client.py

```diff
@@ -516,7 +516,7 @@ def insert(
         )
         return self._execute(statement)
 
-    def load_data(self, path, overwrite=False, partition=None):
+    def load_data(self, path, overwrite=False, partition=None, **kwargs):
```

Comment:
why do we need kwargs? generally we want to explicitly list all parameters if possible

Reply:
as I am adding a test for all backends .. it simplified the test. but I can add specific arguments for each backend.